tests/kubernetes/test_client.py

```diff
@@ -81,3 +85,19 @@ def test_disable_verify_ssl(self):
         else:
             configuration = Configuration()
         assert not configuration.verify_ssl
+
+    def test_api_client_retry_configuration_default(self):
+        conf = initialize_config()
+        api_client_retry_configuration = conf.getjson(
+            "kubernetes", "api_client_retry_configuration", fallback={}
+        )
+
+        assert api_client_retry_configuration == {}
```

Comment:
Remove this test, It's not testing anything other than the upstream `fallback` behaviour.

Reply:
Good catch. Removing the test